Makefile

```diff
@@ -49,7 +49,7 @@ build-api-reference:
 	cd website && poetry run ./build_api_reference.sh
 
 build-docs:
-	cd website && corepack enable && yarn && yarn build
+	cd website && corepack enable && yarn && poetry run yarn build
 
 run-docs: build-api-reference
-	cd website && corepack enable && yarn && yarn start
+	cd website && corepack enable && yarn && poetry run yarn start
```

Comment:
Since the plugin now runs `pydoc-markdown` (and other python tooling) automatically, we need to run the Docusaurus build with the correct venv using `poetry run` (hopefully this is how it works).